ocaml/idl/datamodel.ml

```diff
@@ -2450,12 +2450,15 @@ let vm_migrate_send = call
     ~name: "migrate_send"
     ~in_product_since:rel_tampa
     ~doc: "Migrate the VM to another host.  This can only be called when the specified VM is in the Running state."
-    ~params:[Ref _vm, "vm", "The VM";
-             Map(String,String), "dest", "The result of a Host.migrate_receive call.";
-             Bool, "live", "Live migration";
-             Map (Ref _vdi, Ref _sr), "vdi_map", "Map of source VDI to destination SR";
-             Map (Ref _vif, Ref _network), "vif_map", "Map of source VIF to destination network";
-             Map (String, String), "options", "Other parameters"]
+    ~versioned_params:
+      [{param_type=Ref _vm; param_name="vm"; param_doc="The VM"; param_release=tampa_release; param_default=None};
+       {param_type=Map(String,String); param_name="dest"; param_doc="The result of a Host.migrate_receive call."; param_release=tampa_release;  param_default=None};
+       {param_type=Bool; param_name="live"; param_doc="Live migration"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vdi, Ref _sr); param_name="vdi_map"; param_doc="Map of source VDI to destination SR"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vif, Ref _network); param_name="vif_map"; param_doc="Map of source VIF to destination network"; param_release=tampa_release; param_default=None};
+       {param_type=Map (String, String); param_name="options"; param_doc="Other parameters"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vgpu, Ref _gpu_group); param_name="vgpu_map"; param_doc="Map of source vGPU to destination GPU group"; param_release=vgpu_tech_preview_release; param_default=Some (VMap [])}
```

Comment:
`vgpu_tech_preview_release` was a tech-preview of the first ever vGPU functionality.
We shouldn't use it here. We should use the first release in which this new param will appear: one that has not yet been released.